persistence-cassandra/core/src/test/scala/com/lightbend/lagom/internal/persistence/cassandra/ServiceLocatorHolderSpec.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2016-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+package com.lightbend.lagom.internal.persistence.cassandra
+
+import akka.actor.ActorSystem
+import org.scalatest.{ MustMatchers, WordSpec }
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class ServiceLocatorHolderSpec extends WordSpec with MustMatchers {
+  val system = ActorSystem("test")
+
+  "ServiceLocatorHolder" should {
+    "timeout when no service locator is found" in {
+      val eventually = ServiceLocatorHolder(system).serviceLocatorEventually
+      assertThrows[NoServiceLocatorException](
+        Await.result(eventually, 15.seconds)
```

Comment:
I think I'll take the timeout down to 2 seconds (like it used to be).
I didn't make the timeout configurable when writing the test on purpose. I don't want to make the timeout configurable for the sake of tests.